tonic/benches/decode.rs

```diff
@@ -105,6 +105,10 @@ impl Decoder for MockDecoder {
         buf.advance(self.message_size);
         Ok(Some(out))
     }
+
+    fn buffer_settings(&self) -> tonic::codec::BufferSettings {
+        Default::default()
+    }
```

Comment:
again here we can remove this because we have a default fn on the trait?

Reply:
yep!